bin/cli.js

```diff
@@ -147,18 +148,40 @@ const setRequestHeaders = () => {
   addUserAgentHeader('HubSpot CLI', pkg.version);
 };
 
-const NO_CONFIG_VALIDATION = {
-  init: { skip: true },
-  auth: { skip: true },
+const isTargetedCommand = (options, commandMap) => {
+  const checkCommand = (options, commandMap) => {
+    const currentCommand = options._[0];
+
+    if (!commandMap[currentCommand]) {
+      return false;
+    }
+
+    if (commandMap[currentCommand].target) {
+      return true;
+    }
+
+    const subCommands = commandMap[currentCommand].subCommands || {};
+    if (options._.length > 1) {
+      return checkCommand({ _: options._.slice(1) }, subCommands);
+    }
+
+    return true;
```

Comment:
Should this default to false instead of true?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1310/commits/9b093bb2f1bf781984fe332c411aa08a2bb6ed4b